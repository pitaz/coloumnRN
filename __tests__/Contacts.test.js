import React from 'react';
import {render, cleanup} from '@testing-library/react-native';
import ContactsList from '../src/screens/Contacts';

afterEach(cleanup);

describe('Contacts', () => {
  let rendered;
  beforeEach(() => {
    rendered = render(<ContactsList />);
  });
  it('should match snapshot', () => {
    rendered.toJSON();
    expect(rendered).toMatchSnapshot();
  });

  it('renders list', () => {
    expect(rendered.getByTestId('list')).not.toBeNull();
  });
});
