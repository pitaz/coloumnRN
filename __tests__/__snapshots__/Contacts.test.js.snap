// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contacts should match snapshot 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "padding": 20,
      }
    }
    testID="listWrapper"
  >
    <RCTScrollView
      ItemSeparatorComponent={[Function]}
      ListFooterComponent={[Function]}
      ListHeaderComponent={[Function]}
      disableVirtualization={false}
      getItem={[Function]}
      getItemCount={[Function]}
      horizontal={false}
      initialNumToRender={10}
      keyExtractor={[Function]}
      maxToRenderPerBatch={10}
      onContentSizeChange={[Function]}
      onEndReachedThreshold={2}
      onLayout={[Function]}
      onMomentumScrollEnd={[Function]}
      onScroll={[Function]}
      onScrollBeginDrag={[Function]}
      onScrollEndDrag={[Function]}
      removeClippedSubviews={false}
      renderItem={[Function]}
      scrollEventThrottle={50}
      stickyHeaderIndices={Array []}
      testID="list"
      updateCellsBatchingPeriod={50}
      viewabilityConfigCallbackPairs={Array []}
      windowSize={21}
    >
      <View>
        <View
          onLayout={[Function]}
        >
          <View
            style={
              Object {
                "padding": 15,
              }
            }
          >
            <Text>
              Contacts
            </Text>
          </View>
        </View>
        <View
          onLayout={[Function]}
        >
          <ActivityIndicator
            animating={true}
            color="#000000"
            hidesWhenStopped={true}
            size="large"
          />
        </View>
      </View>
    </RCTScrollView>
  </View>
</RCTSafeAreaView>
`;
